<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Evan Teran</author>
 <class>FunctionFinder::DialogFunctions</class>
 <widget class="QDialog" name="DialogFunctions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Function Finder</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblRegions">
     <property name="text">
      <string>Regions To Search:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtSearch"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Filter</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblResults">
     <property name="text">
      <string>Results:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Start Address</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>End Address</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Size</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Score</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Symbol</string>
      </property>
     </column>	 
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>&amp;Close</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-close"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnHelp">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Help</string>
       </property>
       <property name="icon">
        <iconset theme="help-browser"/>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnGraph">
       <property name="text">
        <string>Graph Selected Function</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFind">
       <property name="text">
        <string>&amp;Find</string>
       </property>
       <property name="icon">
        <iconset theme="edit-find"/>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tableView</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>btnClose</tabstop>
  <tabstop>btnHelp</tabstop>
  <tabstop>btnFind</tabstop>
  <tabstop>txtSearch</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>DialogFunctions</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
